samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_eu.properties

```diff
@@ -269,15 +269,21 @@ datatables_entities=items
 
 ## S2U-17
 t_showStatistics=Estatistikak erakutsi
+statistics_modal_title=Statistics for
```

Comment:
Hey @PabloSanRoman! Could you help me with the additional translations please?
Here are all of them:
```
questions_attempted_help=Number of questions that were either correct or incorrect, but not blank
questions_incorrect_help=A partially incorrect question is considered incorrect
difficulty_help=Calculated from 0 to 100, where 0 represents no difficulty and 100 represents maximum difficulty
excluded_types=The following question types are excluded from the statistics: File Upload, Short Answer/Essay, Student Audio Response, Survey, Survey - Matrix of Choices.
```

Reply:
Hi @stetsche ! Sure, here it is: 

questions_attempted_help=Galdera zuzen edo okerren kopurua, baina ez zuriz.
questions_incorrect_help=Partzialki okerra den galdera bat ez da zuzena.
dificultad_help=0tik 100era kalkulatuta, 0 zailtasuna da eta 100 zailtasuna.
excluido_types=Hurrengo galdera motak estatistiketatik kanpo daude: kargatu fitxategia, idazlana, audio erregistroa, inkesta, inkesta - aukeren matrizea.

Reply:
Great, thanks @PabloSanRoman, I will add them!